CHANGES.rst

```diff
@@ -1,6 +1,12 @@
 Changes
 =======
 
+Version 0.9.3 (2023-12-12)
+--------------------------
+
+- Adds metadata labels to Dockerfile.
+- Changes version of nginx Docker image from 1.19 to 1.25.
```

Comment:
Nginx with capital N?

Reply:
I will change it to NGINX (all uppercase), see for example the main page: https://www.nginx.com/